polaris-for-vscode/.vscodeignore

```diff
@@ -1,9 +1,10 @@
+.turbo
+../**
```

Comment:
Might be helpful to include a comment here since it's resolving a specific edge case. I don't know the ignore pattern off the top, but we could also try to make this self documenting with a negated pattern e.g. `ignore everything that's not polaris-for-vscode`
```
../**
!../polaris-for-vscode
```